tests/run_expired_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+TEMP_DIR=/tmp/temp_aktualizr_expire_repo/$1
+$2/src/aktualizr_repo/aktualizr-repo generate $TEMP_DIR --expires=$1
+$2/src/aktualizr_repo/aktualizr-repo image $TEMP_DIR ./tests/test_data/credentials.zip
+
+cp ./tests/test_data/credentials.zip $TEMP_DIR
+
+PORT=$(python -c 'import socket; s=socket.socket(); s.bind(("", 0)); print(s.getsockname()[1]); s.close()')
+TREEHUB="{\
+  \"oauth2\": {
+    \"server\": \"http://localhost:$PORT\",\
+    \"client_id\": \"54515188-64d1-48bc-bd63-fb707f934fd8\",\
+    \"client_secret\": \"SVgpRfS11q\"\
+  },\
+  \"ostree\": {\
+    \"server\": \"http://localhost:$PORT/\"\
+  }\
+}"
+
+echo $TREEHUB > $TEMP_DIR/treehub.json
+echo "http://localhost:$PORT" > $TEMP_DIR/tufrepo.url
+
+pushd $TEMP_DIR
+zip -r $TEMP_DIR/credentials.zip  treehub.json tufrepo.url
+popd
+./tests/fake_http_server/fake_api_server.py $PORT $TEMP_DIR &
+sleep 2
+
+$2/src/sota_tools/garage-check -j $TEMP_DIR/credentials.zip -r 714581b2ffbbf7a750cb0a210fa7d74fd9128bd627cd4913e365d5bf2f66eba9
+RET=$?
+
+kill %1
+rm -rf $TEMP_DIR/
```

Comment:
For robustness and better error diagnostic on CI:

- running with `set -euo pipefail`  (http://redsymbol.net/articles/unofficial-bash-strict-mode/) and `set -x` for better logs
- using traps for `kill %1` and `rm -rf $TEMP_DIR` (http://redsymbol.net/articles/bash-exit-traps/)
- replacing `rm -rf $TEMP_DIR/` with `rm -rf $TEMP_DIR`. If `TEMP_DIR` is empty (by mistake), it would run `rm -rf /` ([shellcheck](https://www.shellcheck.net/) warned me about this!)